config/generateShellEnv.coffee

```diff
@@ -9,7 +9,7 @@ module.exports.create = (KONFIG, options = {}) ->
   traverse.paths(KONFIG).forEach (path) ->
     node = traverse.get KONFIG, path
     return  if typeof node is 'object'
-    add "KONFIG_#{path.join('_')}".toUpperCase(), node
+    add "KONFIG_#{path.join('_').replace(/\./g, "_")}".toUpperCase(), node
```

Comment:
I really like @cihangir 's idea proposed here #9240 to filter out unused variables. Currently, this script generates > 600 env variables which are mostly unused. I don't use variables generated by this file at all and had to add this logic only because it consumes `goGenerate` object.

`.replace(/\./g, "_")` dosn't change any of existing variables so I think it's safe to left it here.